### ISSUE TITLE
add datadog role for annotations server

### ISSUE THREAD

**Issue by kayiwa** [pulibrary/princeton_ansible#2094]

**Comment by acozine** [comment]
@escowles is the annotations server still in use?

**Comment by escowles** [comment]
@acozine Yes, and we have a few projects wanting to use annotations, so having a production annotation server would be great. @kelynch and @cwulfman are the main folks who would be connecting apps to it.
